src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -646,8 +648,9 @@ enum OpcodeServer
     SMSG_BATTLEFIELD_STATUS_NEEDCONFIRMATION          = 0x59A0,
     SMSG_BATTLEFIELD_STATUS_WAITFORGROUPS             = 0x75A2,
     SMSG_BATTLEFIELD_STATUS_FAILED                    = 0x71A7,
-    SMSG_BATTLEGROUND_INFO_THROTTLED                  = 0x34B2,
+    SMSG_BATTLEGROUND_INFO_THROTTLED                  = 0x14A7, //old 0x34B2,
```

Comment:
Are you certain this is for cata? Looks very simular to a MOP client. How was this obtained along with the others?

Reply:
i had a look into mangos forums on there packet database that was updated 2019 for each client and i checked without any issues. correct me if i am wrong 

https://www.getmangos.eu/wiki/referenceinfo/packetinfo/mangosthreepacket/SMSG_BATTLEGROUND_INFO_THROTTLED-r13303/

MoP 
https://www.getmangos.eu/wiki/referenceinfo/packetinfo/mangosfourpacket/SMSG_BATTLEGROUND_INFO_THROTTLED-r490/

Mop has 0x1E1E